runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -125,6 +125,9 @@
 
         /** Sets expireAfterAccess and expireAfterWrite in seconds. */
         @Default long cacheExpiryTime = Long.MAX_VALUE;
+
+        /** Sets the period of retrieving the latest trim mark in minutes. */
+        @Default long trimMarkSyncPeriod = 10;
```

Comment:
why not private?

Reply:
Now I follow the existing access levels for these parameters. I agree that it should be private as the public getter/setter will be generated.